internal/git/git.go

```diff
@@ -110,6 +110,29 @@ func Log(sha1, sha2 string) (string, error) {
 	return string(outputs), nil
 }
 
+// Fetch a commit from a given remote
+func Fetch(remote, commit string) error {
+	gitcmd := []string{"fetch", remote, commit}
+	cmd := New(gitcmd...)
+	cmd.Stdout = nil
+	cmd.Stderr = nil
+	err := cmd.Run()
+	if err != nil {
+		return errors.Errorf("Can't fetch git commit %s from remote %s", commit, remote)
+	}
+	return nil
+}
+
+// Show all the commits between 2 git commits
+func Show(commit1, commit2 string, reverse bool) {
+	gitcmd := []string{"show"}
+	if reverse {
+		gitcmd = append(gitcmd, "--reverse")
+	}
+	gitcmd = append(gitcmd, fmt.Sprintf("%s..%s", commit1, commit2))
+	New(gitcmd...).Run()
+}
+
```

Comment:
```suggestion

// GetLocalRemotes returns a string of local remote names and URLs
func GetLocalRemotes() (string, error) {
        cmd := New("remote", "-v")
        cmd.Stdout = nil
        remotes, err := cmd.Output()
        if err != nil {
                return "", err
        }

        return string(remotes), nil
}

```

Reply:
These changes look at the MR's project URL and maps that to name of the fetch repo in 'git remote -v'.  That way 

lab mr show 9 --patch 

and

lab mr show origin 9 --patch

both work.